homeassistant/components/shelly/sensor.py

```diff
@@ -358,6 +358,7 @@ def __init__(
         translation_key="lamp_life",
         value=get_shelly_air_lamp_life,
         suggested_display_precision=1,
+        # Deprecated, remove in 2025.10
```

Comment:
This

![obraz](https://github.com/user-attachments/assets/7a0629fd-39c4-498c-8983-e9ae024a2c19)

is not entirely true for this attribute because there is no corresponding entity for it.

Reply:
Correct.
Should we offer the template as I first supposed to for this ?

Reply:
I think it would be better to do something like this:

The extra state attributes has been deprecated and will be removed, please use the following instead:
| Extra state attribute | Use instead     |
|-----------------------|-----------------|
| Gas detected           | Gas detected entity   |
| Gas test status         | Gas self test entity  |
| Air total work hours           | `{{ ((100 - states('sensor.shelly_air_lamp_life') \| float) * 90) \| round(1) }}`  |